bundles/org.openhab.binding.icalendar/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -99,4 +99,113 @@
 		<description>End of the next event in calendar</description>
 		<state readOnly="true"/>
 	</channel-type>
+
+	<channel-type id="result_start">
+		<item-type>DateTime</item-type>
+		<label>Start of result</label>
+		<description>Start of the found result in calendar</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="result_end">
+		<item-type>DateTime</item-type>
+		<label>End of result</label>
+		<description>End of the found result in calendar</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="result_title">
+		<item-type>String</item-type>
+		<label>Title of result</label>
+		<description>Title of the found result in calendar</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-group-type id="result">
+		<label>Result Event</label>
+		<description>A resulting event found by filter</description>
+		<channels>
+			<channel typeId="result_start" id="begin"/>
+			<channel typeId="result_end" id="end"/>
+			<channel typeId="result_title" id="title"/>
+		</channels>
+	</channel-group-type>
+
+
+	<thing-type id="eventfilter">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="calendar"/>
+		</supported-bridge-type-refs>
+		<label>Event filter</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
OK